internal/query/sms_test.go

```diff
@@ -127,9 +127,10 @@ func Test_SMSConfigssPrepare(t *testing.T) {
 						State:         domain.SMSConfigStateInactive,
 						Sequence:      20211109,
 						TwilioConfig: &Twilio{
-							SID:          "sid",
-							Token:        &crypto.CryptoValue{},
-							SenderNumber: "sender-number",
+							SID:              "sid",
+							Token:            &crypto.CryptoValue{},
+							SenderNumber:     "sender-number",
+							VerifyServiceSID: "",
```

Comment:
Unless this is a code-style thing, you shouldn't need to explicitly set an empty string for a non-pointer string, it will default to empty.

Some codebases have style guides around explicit instantiation, so ignore this if that is the case here.